src/app/profile/profile.page.ts

```diff
@@ -55,13 +53,26 @@ export class ProfilePage {
         control.disabled =true;
         if (control.type === 'select') {
           control.type = 'text';
+        }else if(control.type === 'checkbox'){
+          control.type = 'chip'
+        }else{
+          control.type = 'text';
         }
         control.value = typeof (value) === 'string' ? String(value) : value?.label;
+        control.validators = false
+        control.label = this.capitalizeLabel(control.name);
       }
     });
   }
 
   goBack() {
     this.navCtrl.back();
   }
+
+  capitalizeLabel(label: string): string {
```

Comment:
are you capitalizing the whole word or just the initial letter? if so then please change the function to something that suits the function